runtime/sys.c

```diff
@@ -586,7 +586,7 @@ CAMLprim value caml_sys_random_seed (value unit)
   /* If the read from /dev/urandom fully succeeded, we now have 96 bits
      of good random data and can stop here.  Otherwise, complement
      whatever we got (probably nothing) with some not-very-random data. */
-  if (n < 12) {
+  if (n < 11) {
```

Comment:
Note: after thinking harder about this, I think that changing `n < 12` to `n < 11` is not the best way to do this change. (If we get exactly 11 bytes from /dev/urandom, we do want to add some non-random seeds; currently the after-PR implementation will lack a bit of entropy in this weird corner case that will probably not occur in practice.)

I proposed a refactoring of the upstream function in https://github.com/ocaml/ocaml/pull/10472 that would have made it easier to correctly extend the function.